mix.exs

```diff
@@ -30,6 +31,12 @@ defmodule Phoenix.LiveDashboard.MixProject do
     ]
   end
 
+  defp aliases do
+    [
+      dev: "run --no-halt dev.exs"
```

Comment:
Beautiful! While at it, could you add the following alias too? It's a default in new phoenix apps and was meaning to add it myself but forgot about it :)

```suggestion
      setup: ["deps.get", "cmd npm install --prefix assets"],
      dev: "run --no-halt dev.exs"
```